website/src/assets/icons/diamond.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="currentColor" class="bi bi-diamond-fill" viewBox="0 0 16 16">
+  <path fill-rule="evenodd" d="M6.95.435c.58-.58 1.52-.58 2.1 0l6.515 6.516c.58.58.58 1.519 0 2.098L9.05 15.565c-.58.58-1.519.58-2.098 0L.435 9.05a1.482 1.482 0 0 1 0-2.098L6.95.435z"/>
+</svg>
```

Comment:
![image](https://user-images.githubusercontent.com/2106987/118127622-881b8e00-b42c-11eb-8056-f531549e8cd4.png)

Hi, we need to have a blank line at the end of files.